Python/cpuinfo.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Naive CPU SIMD features detection.
+ *
+ * See Modules/black2module.c.
+ */
+
+#include "Python.h"
+#include "pycore_cpuinfo.h"
+
+#include <stdbool.h>
+
+#if defined(__x86_64__) && defined(__GNUC__)
```

Comment:
Is this the unique header file of the GNU-collection?

Reply:
That's the only one containing __cpuid_count.